dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -8,50 +8,14 @@
     vertical-align: top;
   }
 
-%p
-  We’re excited to continue to work together to bring computer science to your classroom.
-  - if @workshop.local_summer? && @is_reminder
-    Please take a moment to complete the pre-course survey and review the
-    following logistical details about your workshop.
-  If you have any questions about your workshop, reach out to your workshop organizer directly:
-  = @workshop.organizer.name
-  at
```

Comment:
It looks like some of this stuff should remain. This line and line 12 are common, outside the local_summer conditional.